src/Components/Layout/StockLayout.jsx

```diff
@@ -1,17 +1,17 @@
 import { Layout } from "antd";
 import React from "react";
```

Comment:
React import should be at the top along with all other internal imports, then a space and all external imports, another space and finally css imports